api/organisations/tests/test_views.py

```diff
@@ -173,13 +173,17 @@ def test_should_return_all_invites_and_can_resend(self):
         assert invite_list_response.status_code == status.HTTP_200_OK
         assert invite_resend_response.status_code == status.HTTP_200_OK
 
-    def test_can_remove_a_user_from_an_organisation(self):
+    def test_remove_user_from_an_organisation_also_removes_from_group(self):
```

Comment:
I think this should be a separate test - one that also checks that other users are not removed from the group at the same time.

Reply:
I can think we can add that in the same test?